File: A2Z DSA/Learn the basics/Program1.cpp
// Check whether someone is an adult or not
// Name = Indrajeet Mondal; Date = 3rd July 2024
// SourceCode

#include <bits/stdc++.h>
using namespace std;

int main()
{
    int age;

    cout << "Enter your age:- ";
    cin >> age;

    if (age >=  18)
    {
        cout << "You are an adult." << endl;
    }
    else
    {
        cout << "You are not an adult." << endl;
    }

    return 0;
}

File: A2Z DSA/Learn the basics/Know Basic Maths/Program1.py
# Count no. of digits in the given number (Brute-Force Approach)
# Name = Indrajeet Mondal; Date = 30th July 2024
# SourceCode

x = int(input("Enter a number:- "))
count = 0
z = x

while z > 0:
    z = z // 10
    count += 1

print(f"The number of digits in {x} is {count}.")
